octoprint_mrbeam/templates/laserhead_changed.jinja2

```diff
@@ -1,19 +1,87 @@
+    {# Not supported laserhead  - START #}
+    <div class="modal-body" data-bind="visible: !laserhead_model_supported">
         <div>
-            <img loading="lazy" src="/plugin/mrbeam/static/img/welcome_dialog/start.png">
-            <h4> {{ _("Your new Mr Beam dreamcut [S] Laserhead has been successfully installed.") }}</h4>
-            <p> {{ _("With this Laserhead you can work now even faster.") }}</p>
-            <p> {{ _("This means that we also update the default material settings to the new laserhead as long it is installed to provide you the best possible support. However, we do not update your custom materials, this will need your personal testing and finetuning.") }}</p>
-            <p> {{ _("Read about the changes you can expect with the new and improved MrBeam II dreamcut [S] Laserhead.") }} <a target="_blank" href="{{ _("https://support.mr-beam.org/en/support/solutions/articles/43000640531") }}"> » {{ _("Read more") }} </a></p>
+            <img loading="lazy" src="/plugin/mrbeam/static/img/Welcome.png">
+            <h4> {{ _("We are verv sorry but the installed Laser head will not work with vour device.") }}</h4>
+            <p> {{ _("Every device older than the dreamcut row will not support the new and advanced Laser head properly. There are just too many improvements that are missing in the earlier states of our model rows like Mr Beam and Mr Beam II. Please contact our support tor further steps.") }}</p>
         </div>
     </div>
+    {# Not supported laserhead  - END #}
+
+
+    {# Steps for laserhead S  - START #}
+    <div class="modal-body" data-bind="visible: show_step_for_laserhead_s(1)">
+        <div>
+            <img loading="lazy" src="/plugin/mrbeam/static/img/Welcome.png">
+            <h4> {{ _("Your Mr Beam dreamcut [S] Laser head has been successfully installed.") }}</h4>
+            <p> {{ _("With this Laser head you can work now even faster.") }}</p>
+            <p> {{ _("This means that we also update the default material settings to the new laser head as long it is installed to provide you the best possible support. However, we do not update your custom materials, this will need your personal testing and finetuning.") }}</p>
+            <p> {{ _("Read about the changes you can expect with the MrBeam II dreamcut [S] Laser head.") }} <a
+                    target="_blank"
+                    href="{{ _("https://support.mr-beam.org/en") }}/support/solutions/articles/43000640531">
+                » {{ _("Read more") }} </a></p>
+        </div>
+    </div>
+    {# Steps for laserhead S  - END #}
+
+    {# Steps for laserhead x  - START #}
+    <div class="modal-body" data-bind="visible: show_step_for_laserhead_x(1)">
+        <div>
+            <img loading="lazy" src="/plugin/mrbeam/static/img/Welcome.png">
+            <h4> {{ _("Your Mr Beam II dreamcut [X] laser head has been successfully connected.") }}</h4>
+            <p> {{ _(" You can now work even faster with this laser head.") }}</p>
```

Comment:
Would it make sense to use the same wording here as above?

Reply:
Daniel wanted it like this